machineEnvironment.sh

```diff
@@ -35,6 +35,7 @@ queue=""         # standard queue to submit jobs to
 nthreads=""      # number of threads to use for parallel builds
 mpilaunch=""     # command to launch an MPI executable (e.g. aprun)
 installdir=""    # directory where libraries are installed
+container_engine=""  # Engine for running containers, e.g. docker, sarus, singuilarity
```

Comment:
It seems like terms runtime and engine are somewhat subtle distinctions:
https://developers.redhat.com/blog/2018/02/22/container-terminology-practical-introduction/#h.6yt1ex5wfo3l
I was thinking 'engine' because docker/sarus could potentially used for other commands besides just running a container. I am happy to change it to container_runtime if that is more correct and/or preferred.

Reply:
You are right. Engine is the way to go.